binding.gyp

```diff
@@ -30,38 +30,14 @@
                 "cflags": [
                     "-std=gnu99",
                 ],
-                "conditions": [
-                    [ "system_dc_core == 'false'", {
```

Comment:
This should be replaced with
```
"system_dc_core == ''"
```
in the end because `echo $SYSTEM_DC_CORE` produces an empty string if the variable is undefined.